### DISCUSSION TITLE
Issue with NPM Registry downloads on Sept 12/13 + more days

### DISCUSSION THREAD

**Discussion by bernardobridge** [npm/feedback#1001] [Registry]
Hello,

I noticed one of our packages had substantially less downloads this week. 

At first I wondered if it had to do with Node 16 EOL, but in digging deeper using https://npm-stat.com/ we had 0 reported downloads on September 12 and 13. And potentially 14th showing less than norm.

Still might be related, but obviously 0 downloads is wrong. Once I saw that stat I looked up other major packages (e.g. lodash, got, zod, etc). They all have the same dip and 0 downloads on those days.

Was this a known issue? Is it reported anywhere?

Thank you

**Comment by capricorn86** [comment]
I have also noticed a drop in Weekly Downloads. It has gone down about 50% for my package.

**Comment by bernardobridge** [comment]
It's happening again - for the last 2 days.

**Comment by iambumblehead** [comment]
npm download stats are broken for all packages https://api.npmjs.org/downloads/range/2023-09-10:2023-09-19/uuid

**Comment by iambumblehead** [comment]
related discussions,
 * https://news.ycombinator.com/item?id=37586956
 * https://old.reddit.com/r/programming/comments/16mna5z/whats_going_on_in_the_last_week_that_has_caused_a/k195ovg/

**Comment by bernardobridge** [comment]
This seems to be happening again. Is it possible to get an official response on this?
